OrbitService/OrbitAsioServer.cpp

```diff
@@ -66,7 +68,18 @@ void OrbitAsioServer::StartCapture(int32_t pid) {
   }
 
   LOG("Starting capture");
-  tracing_handler_.Start(pid, selected_functions_);
+
+  CaptureOptions capture_options = capture_options_;
+  capture_options.set_pid(pid);
+  for (const std::shared_ptr<Function>& function : selected_functions_) {
+    CaptureOptions::InstrumentedFunction* instrumented_function =
+        capture_options.add_instrumented_functions();
+    instrumented_function->set_binary_path(function->GetLoadedModulePath());
+    instrumented_function->set_file_offset(function->Offset());
+    instrumented_function->set_virtual_address(function->GetVirtualAddress());
```

Comment:
ok, It is an absolute address, can you rename it to absolute_address? virtual address in confusing in elf-context it means different thing.

Reply:
Done.